### PR TITLE
Move Path check into switch case

### PR DESCRIPTION
**Pull request by Nalum**
Fix: #245

### CODE REVIEW

File: controllers/imageupdateautomation_controller.go [hunk 1]
```diff
@@ -269,6 +259,16 @@ func (r *ImageUpdateAutomationReconciler) Reconcile(ctx context.Context, req ctr
...
 		}
 
+		manifestsPath := tmp
+		if auto.Spec.Update.Path != "" {
+			tracelog.Info("adjusting update path according to .spec.update.path", "base", tmp, "spec-path", auto.Spec.Update.Path)
+			if p, err := securejoin.SecureJoin(tmp, auto.Spec.Update.Path); err != nil {
+				return failWithError(err)
+			} else {
+				manifestsPath = p
+			}
```
Comment (pjbgf) [inline controllers/imageupdateautomation_controller.go:269]: nit: `else` is redundant when followed by a `return`.
```suggestion
			p, err := securejoin.SecureJoin(tmp, auto.Spec.Update.Path);
			if err != nil {
				return failWithError(err)
			}
			manifestsPath = p
```

Comment (Nalum) [reply controllers/imageupdateautomation_controller.go]: Thanks @pjbgf :+1:

**Review by pjbgf — APPROVED** [review]
LGTM

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @Nalum 🏅

**Pull request merged**